includes/core/classes/class-event-query.php

```diff
@@ -189,8 +189,9 @@ public function prepare_event_query_before_execution( WP_Query $query ): void {
 					if ( $page->id === $query->queried_object_id ) {
 						$query->set( 'post_type', 'gatherpress_event' );
```

Comment:
Maybe bring this down with the other `->set` on line 194 to keep like things together.